include/ebpf_nethooks.h

```diff
@@ -172,3 +169,61 @@ typedef struct bpf_sock_addr
  */
 typedef int
 sock_addr_hook_t(bpf_sock_addr_t* context);
+
+typedef enum _bpf_sock_op_type
+{
+    BPF_SOCK_OPS_ACTIVE_ESTABLISHED_CB,  ///< Calls BPF program when an
+                                         ///< active (outbound) connection is
+                                         ///< established
+                                         ///<
```

Comment:
the `///<` format won't work, move any multi-line comments to lines before and use `/** @brief multi line comment here */` like I did in PR #940